mtp_api/apps/core/tests/utils.py

```diff
@@ -20,4 +22,14 @@ def make_test_oauth_applications():
         authorization_grant_type='password',
         client_secret='cashbook',
         name='cashbook',
-        user=User.objects.first())
+        user=User.objects.first()
+    )
+
+    Application.objects.get_or_create(
+        client_id='noms-admin',
```

Comment:
Yeah… And that's not exactly what the role is, either. `prisoner-location-admin`?